src/shared/cache/GenomeNexusEnrichment.ts

```diff
@@ -37,7 +36,14 @@ export function fetch(queries: Mutation[]):Promise<VariantAnnotationEnriched[]>
 
 export default class GenomeNexusCache extends LazyMobXCache<VariantAnnotationEnriched, Mutation> {
     constructor() {
-        super((m:Mutation) => (generateGenomeNexusQuery(m)), // queryToKey
+        super((m:Mutation) => {
+                  const genomicLocation = extractGenomicLocation(m);
+                  if (genomicLocation) {
+                    return genomicLocationString(genomicLocation); // queryToKey
+                  } else {
+                      return "";
```

Comment:
@onursumer can u check if this is correct, not sure what key value should be returned for the cache if the input format is incorrect.

Reply:
@inodb Actually, not sure what to do in case of a mutation with invalid genomic location. Maybe we should filter out mutations with invalid genomic locations before invoking any cache method?